src/webhook/test_identity_validation.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+import json
+
+from webhook.identity_validation import get_response
+
+ADMIN_GROUPS = [
+    "osd-sre-admins"
+]
+
+
+def create_request(username, groups, identityName, providerName):
+    class FakeRequest(object):
+        def __init__(self):
+            self.json = {
+                "request": {
+                    "uid": "testuser",
+                    "operation": "UPDATE",
```

Comment:
Should we write test to iterate over the operations we care about and just make sure we don't have regressions?  Could verify CREATE, UPDATE, and DELETE.

Reply:
No, we can only do it in e2e tests, this is only unit test, the validation func doesn't check whether it's "CREATE" or "UPDATE", the verbs are configured in `ValidatingWebhookConfiguration`.